extensions/notebook/src/book/bookModel.ts

```diff
@@ -102,6 +104,12 @@ export class BookModel {
 		if (this.isNotebook) {
 			return undefined;
 		}
+		let notebookConfig = vscode.workspace.getConfiguration(constants.notebookConfigKey);
+		let maxExpandedItems = notebookConfig[constants.maxExpandedBookItems];
+		let collapsibleState = vscode.TreeItemCollapsibleState.Expanded;
+		if (this.bookQuantity > maxExpandedItems) {
```

Comment:
If the Max Expanded Book Items is 2 and you opened folder with 5 books, all of them will be collapsed? It'll not expand 2 and collapse the rest. Is that the expected behavior?

Reply:
My 2 cents - that seems reasonable to me. It seems kind of weird to only open a couple folders - how would we know which are the "right" ones to choose? The main point of this setting I assume is so that people don't have to go and collapse the tree to see the other books in the folder.